src/Compilers/CSharp/Test/CommandLine/app.config

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <appSettings>
+    <add key="xunit.useAppDomain" value="true"/>
+    <add key="xunit.diagnosticMessages" value="false"/>
```

Comment:
Per http://xunit.github.io/docs/configuring-with-xml.html, these first two are the default? Can we delete them?